server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/functions.js

```diff
@@ -268,6 +268,19 @@ function createEmptyRow(col, msg) {
   return '<td class="center" colspan="' + col + '"><i>' + msg + '</i></td>';
 }
 
+/**
+ * Performs an ajax reload for the given DataTable
+ *
+ * @param {DataTable} table DataTable to perform an ajax reload on
+ */
+function ajaxReloadTable(table) {
+  if (table) {
+    table.ajax.reload(null, false); // user paging is not reset on reload
+  } else {
+    console.error('There was an error reloading the given table');
```

Comment:
This is nice. You could log what was passed in.
```suggestion
    console.error('There was an error reloading the given table: ' + table);
```

Reply:
I just did a quick test in the console and it would print out like this: `There was an error reloading the given table: [object Object]` since `table` is a DataTable object.

Reply:
Ah yeah. I am not sure what we could print cause i think that means it doesn't exist.

Reply:
Yea not too sure. I've been looking around and haven't found a good solution so I'm thinking I'll merge this as is and we can follow up at somepoint if needed.